GOVERNANCE.md

```diff
@@ -0,0 +1,66 @@
+### Documentation Working Group
+
+The Node.js Documentation group is jointly governed by a Working Group (WG) which is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
+
+* Technical direction
+* Project governance and process (including this policy)
+* Contribution policy
+* GitHub repository hosting
+* Conduct guidelines
+* Maintaining the list of additional Collaborators
+
+For the current list of WG members, see the project
+README.md.
+
+### Collaborators
```

Comment:
(Moved previous comment to correct section.) 

I'd say this is different, as the CONTRIBUTING.md talks about creation documentation, whereas this refers to the meta content of the WG.